README.md

```diff
@@ -110,6 +110,7 @@ This is the list of all environment variables including brief description
 | ISSUANCE_DID_POLICY            | y        | EXISTING_OR_NEW                                                | DID and either create or use existing: [CREATE_NEW, FIND_EXIStING, EXISTING_OR_NEW, did:somedid] |
 | ISSUANCE_SCHEMA_POLICY         | y        | EXISTING_OR_NEW                                                | Same as above but for credential schema                                                          |
 | ISSUANCE_CRED_DEF_POLICY       | y        | EXISTING_OR_NEW                                                | Same as above but this is for credential definitions                                             |
+| DEMO_MODE                      | y        | true                                                           | Wheater end-user is allowed to hit /reset endpoint or not
```

Comment:
did you mean `Whether` instead of `Whatever` ?

Reply:
it's a typo, thanks, updated